wikipediapreview.php

```diff
@@ -14,25 +14,35 @@
  */
 
 DEFINE( 'WIKIPEDIA_PREVIEW_PLUGIN_VERSION', '1.0.6' );
+DEFINE( 'STYLESHEET_MEDIA_TYPE', 'all' );
```

Comment:
I think this name is too generic for a global constant. It risks clashing with other code. It could start with `WIKIPEDIA_PREVIEW_` like the one above.